lib/src/bbob_parser/lexer.dart

```diff
@@ -157,32 +160,76 @@ class Lexer {
         }
       }
 
-      return !isEqual && !isWhiteSpace;
+      return !isWhiteSpace;
     }
 
-    void _nextAttr() {
-      final attrStr = attrCharGrabber.grabWhile(validAttr);
-      final currChar = attrCharGrabber.current;
+    // Represents: https://github.com/JiLiZART/BBob/blob/99c629e66678e40cec7b924eda2f1bfe0d46b941/packages/bbob-parser/src/lexer.ts#L104
```

Comment:
DO use /// doc comments to document members and types
https://dart.dev/effective-dart/documentation#do-use-doc-comments-to-document-members-and-types

```suggestion
    /// Returns next `TagState` by keeping grabbing chracters
    ///
    /// Represents: https://github.com/JiLiZART/BBob/blob/99c629e66678e40cec7b924eda2f1bfe0d46b941/packages/bbob-parser/src/lexer.ts#L104
```

Reply:
```suggestion
    /// Processes part of a tag and returns a state that should be used for the next iteration.
    ///
    /// Represents: https://github.com/JiLiZART/BBob/blob/99c629e66678e40cec7b924eda2f1bfe0d46b941/packages/bbob-parser/src/lexer.ts#L104
```